### PR TITLE
Add new badges.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,14 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
...
+### Support the project
+
+[![Open Collective](https://img.shields.io/badge/Open%20Collective-sponsor-7eadf1?logo=open%20collective&logoColor=7eadf1&labelColor=555555)](https://opencollective.com/yiisoft)
+
+### Follow update
+
+[![Yii2](https://img.shields.io/badge/Powered_by-Yii_Framework-green.svg?style=flat)](https://www.yiiframework.com/)
+[![Twitter Follow](https://img.shields.io/badge/twitter-follow-1DA1F2?logo=twitter&logoColor=1DA1F2&labelColor=555555?style=flat)](https://twitter.com/yiiframework)
+[![Telegram](https://img.shields.io/badge/telegram-join-1DA1F2?style=flat&logo=telegram)](https://t.me/yii3ru)
+[![Facebool](https://img.shields.io/badge/facebook-join-1DA1F2?style=flat&logo=facebook&logoColor=ffffff)](https://www.facebook.com/groups/yiitalk)
```
Comment (viktorprogger) [inline README.md:62]:
```suggestion
[![Facebook](https://img.shields.io/badge/facebook-join-1DA1F2?style=flat&logo=facebook&logoColor=ffffff)](https://www.facebook.com/groups/yiitalk)
```

Comment (terabytesoftw) [reply README.md]: Fixed.

**Review by xepozz — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -49,3 +49,15 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
 ```php
 ./vendor/bin/psalm
 ```
+
+### Support the project
+
+[![Open Collective](https://img.shields.io/badge/Open%20Collective-sponsor-7eadf1?logo=open%20collective&logoColor=7eadf1&labelColor=555555)](https://opencollective.com/yiisoft)
+
+### Follow update
```
Comment (samdark) [inline README.md:57]:
```suggestion
### Follow updates
```

File: README.md [hunk 3]
```diff
@@ -49,3 +49,15 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
...
+
+[![Yii2](https://img.shields.io/badge/Powered_by-Yii_Framework-green.svg?style=flat)](https://www.yiiframework.com/)
+[![Twitter Follow](https://img.shields.io/badge/twitter-follow-1DA1F2?logo=twitter&logoColor=1DA1F2&labelColor=555555?style=flat)](https://twitter.com/yiiframework)
+[![Telegram](https://img.shields.io/badge/telegram-join-1DA1F2?style=flat&logo=telegram)](https://t.me/yii3ru)
+[![Facebook](https://img.shields.io/badge/facebook-join-1DA1F2?style=flat&logo=facebook&logoColor=ffffff)](https://www.facebook.com/groups/yiitalk)
+[![Slack](https://img.shields.io/badge/slack-join-1DA1F2?style=flat&logo=slack)](https://join.slack.com/t/yii/shared_invite/enQtMzQ4MDExMDcyNTk2LTc0NDQ2ZTZhNjkzZDgwYjE4YjZlNGQxZjFmZDBjZTU3NjViMDE4ZTMxNDRkZjVlNmM1ZTA1ODVmZGUwY2U3NDA)
```
Comment (samdark) [inline README.md:63]: That is potentially a problem. Sometimes slack resets the token so it changes. If we'll list it here, we'll have to update it everywhere.

Comment (terabytesoftw) [reply README.md:63]: no direct link to slack?

File: README.md (hunk 2 above)
Comment (terabytesoftw) [reply README.md]: Fixed.

File: README.md [hunk 4]
```diff
@@ -49,3 +49,15 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
...
 ./vendor/bin/psalm
 ```
+
+### Support the project
+
+[![Open Collective](https://img.shields.io/badge/Open%20Collective-sponsor-7eadf1?logo=open%20collective&logoColor=7eadf1&labelColor=555555)](https://opencollective.com/yiisoft)
+
+### Follow updates
+
+[![Yii2](https://img.shields.io/badge/Powered_by-Yii_Framework-green.svg?style=flat)](https://www.yiiframework.com/)
```
Comment (roxblnfk) [inline README.md:59]: Yii2?

File: README.md (hunk 3 above)
Comment (samdark) [reply README.md:63]: Nope. We can likely create a redirect-URL at yiiframework.com...

Comment (samdark) [reply README.md:63]: I think we can link to https://www.yiiframework.com/chat

File: README.md (hunk 4 above)
Comment (samdark) [reply README.md:59]: Nope :)

File: README.md (hunk 3 above)
Comment (xepozz) [reply README.md:63]: Just place https://yii.slack.com/archives/

Comment (samdark) [reply README.md:63]: @xepozz won't work. It's not an invitation link.

Comment (samdark) [reply README.md:63]: Now there's way to have a good link. Here it is: https://yiiframework.com/go/slack

File: README.md [hunk 5]
```diff
@@ -49,3 +49,15 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
...
+[![Twitter Follow](https://img.shields.io/badge/twitter-follow-1DA1F2?logo=twitter&logoColor=1DA1F2&labelColor=555555?style=flat)](https://twitter.com/yiiframework)
+[![Telegram](https://img.shields.io/badge/telegram-join-1DA1F2?style=flat&logo=telegram)](https://t.me/yii3ru)
+[![Facebook](https://img.shields.io/badge/facebook-join-1DA1F2?style=flat&logo=facebook&logoColor=ffffff)](https://www.facebook.com/groups/yiitalk)
+[![Slack](https://img.shields.io/badge/slack-join-1DA1F2?style=flat&logo=slack)](https://join.slack.com/t/yii/shared_invite/enQtMzQ4MDExMDcyNTk2LTc0NDQ2ZTZhNjkzZDgwYjE4YjZlNGQxZjFmZDBjZTU3NjViMDE4ZTMxNDRkZjVlNmM1ZTA1ODVmZGUwY2U3NDA)
```
Comment (samdark) [inline README.md:63]:
```suggestion
[![Slack](https://img.shields.io/badge/slack-join-1DA1F2?style=flat&logo=slack)](https://yiiframework.com/go/slack)
```

File: README.md (hunk 4 above)
Comment (samdark) [inline README.md:59]:
```suggestion
[![Official website](https://img.shields.io/badge/Powered_by-Yii_Framework-green.svg?style=flat)](https://www.yiiframework.com/)
```

**Review by samdark — APPROVED** [review]

**Pull request merged**